training_config/transformer_squad.bert-large.jsonnet

```diff
@@ -0,0 +1,60 @@
+local transformer_model = "bert-large-cased";
+local epochs = 3;
+local batch_size = 8;
+
```

Comment:
I'd strongly suggest deduplicating across these configs so we can see what's changing at a glance. See https://github.com/allenai/allennlp/blob/master/allennlp/tests/fixtures/language_model/experiment_unidirectional.jsonnet and the other files for an example. It's pretty straightforward, but I would suggest sanity checking your generated config afterwards.

Reply:
be5108caba242413e74c781474354179d3fd52ae

In a way, this simultaneously demonstrates the power of jsonnet, and why I think it's a bad idea anyways. It's really cool that this works. And it's not clear that introducing another programming language is the best way to achieve it.